README.md

```diff
@@ -21,23 +21,21 @@ cd bookkeeper-visual-manager
 mvn clean install -DskipTests
 ~~~~
 
-To pass the Metadata Service URI connection string you have to set the System property
-`bookkeeper.visual.manager.metadataServiceUri`.
+To pass the Metadata Service URI connection string you have to set the system property
+`bookkeeper.visual.manager.metadataServiceUri` or run the provided script in bin
+folder.
 
 ~~~~
 BVM_PORT=8080
 BVM_SERVICEURI=zk+null://localhost:2181/ledgers
 
-# System property configuration
-mvn jetty:run \
-    -Dbookkeeper.visual.manager.metadataServiceUri=$BVM_SERVICEURI \
-    -Djetty.http.port=$BVM_PORT
+./bin/run.sh $BVM_SERVICEURI $BVM_PORT
```

Comment:
What about run_dev.sh ?
'run.sh' sounds like a production script to me

Reply:
+1, especially since we required maven to run this script

Reply:
You renamed the file, we have to fix this description